Fastfile

```diff
@@ -26,6 +26,15 @@ default_platform :ios
     _buildAndDeployToHockey()
   end
 
+  lane :deploy_to_test_flight do
+    _setup()
+    scan
+    icon_overlay(version: get_version_number)
```

Comment:
Good point. Should we have a separate `deploy_to_app_store` lane on get rid of the icon overlay altogether when submitting to iTC?